src/dms/core/EdFi.DataManagementService.Core/Middleware/ProvideApiSchemaMiddleware.cs

```diff
@@ -7,6 +7,7 @@
 using System.Text.Json.Nodes;
 using EdFi.DataManagementService.Core.ApiSchema;
 using EdFi.DataManagementService.Core.ApiSchema.Helpers;
+using EdFi.DataManagementService.Core.Backend;
```

Comment:
@semalaiappan 
Note: After merging the changes from DMS-586, this file requires no further updates or changes.